shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -78,13 +78,12 @@ - (instancetype)initWithEngine:(FlutterEngine*)engine
   if (self) {
     _viewOpaque = YES;
     _engine.reset([engine retain]);
-    _engineNeedsLaunch = NO;
+    _engineNeedsLaunch = YES;
     _flutterView.reset([[FlutterView alloc] initWithDelegate:_engine opaque:self.isViewOpaque]);
     _weakFactory = std::make_unique<fml::WeakPtrFactory<FlutterViewController>>(self);
     _ongoingTouches = [[NSMutableSet alloc] init];
 
     [self performCommonViewControllerInitialization];
-    [engine setViewController:self];
```

Comment:
So this line here is what's causing the view to load.

Reply:
Yes. That right.